src/entrypoints/core.ts

```diff
@@ -89,13 +93,10 @@ window.hassConnection.then(({ conn }) => {
   subscribeThemes(conn, noop);
   subscribeUser(conn, noop);
 
-  if (location.pathname === "/" || location.pathname.startsWith("/lovelace/")) {
-    (window as WindowWithLovelaceProm).llConfProm = fetchConfig(
-      conn,
-      null,
-      false
-    );
-  }
+  (window as WindowWithLovelaceProm).llConfProm = fetchResourcesAndConfig(
```

Comment:
We shouldn't do this in 1 function. This should be 2 functions, as they always should run in parallel. I want to start processing either the resources or the config while the other has not returned yet. By waiting for both, we're now stalling for no reason.